### ISSUE TITLE
resizing is bad while wobbling

### ISSUE THREAD

**Issue by linuxuser1337** [WayfireWM/wayfire#486]
When I resize a window it is hard to do if windows are wobbly.
Is it possible to have wobbly windows, but not while resizing?

**Comment by ammen99** [comment]
Yes it would be possible, we just haven't implemented an option for that yet.
